packages/cardpay-cli/rewards/claimable-proofs.ts

```diff
@@ -88,6 +88,9 @@ function displayProof(o: WithSymbol<Proof> | WithSymbol<ClaimableProof>, rewardP
       gasFees: ${
         rewardPool.isClaimableProof(o) ? fromWei(o.gasEstimate.amount) + ' ' + o.tokenSymbol : 'No Gas Estimates'
       }
+      explanationTemplate: ${o.explanationTemplate ?? '**Not specified**'} 
+      explanationData: 
+         ${JSON.stringify(o.explanationData, null, 4) ?? '**Not specified**'} 
```

Comment:
Having `**Not specified**` with the asterisks is proposital as some sort of markdown? Will the rewards program have the ability to add Markdown to their explanations?

Reply:
This fn only prints to console, so it doesn't matter that much the format used. Maybe the name `displayProof` does not convey that properly. I would suggest something like `logProof`.

Reply:
I can remove the asterisk anyway. I renamed displayProof to logProof